### DISCUSSION TITLE
code server deployed on heroku.But heroku not allowing to see inside container.

### DISCUSSION THREAD

**Discussion by MuhammadAli-dev** [coder/code-server#5567] [Q&A]
I cannot ls or cd anything.Help me please

![1](https://user-images.githubusercontent.com/64697202/190851854-060ddb02-f72d-4320-8f2d-96ff2d36fa0b.png)
![2](https://user-images.githubusercontent.com/64697202/190851857-836e83d4-9149-4a9b-b23f-dedb50a39587.png)
![3](https://user-images.githubusercontent.com/64697202/190851862-e42c1a70-d06f-4f25-b736-0f06a29e7c71.png)
![4](https://user-images.githubusercontent.com/64697202/190851873-61406128-63bb-4b85-8c07-e7b68963d1e2.png)
![5](https://user-images.githubusercontent.com/64697202/190851878-00a36459-7654-44ca-a4be-9445b5c625aa.png)

**Comment by jsjoeio** [comment]
It seems like that console only allows you to run `heroku run` containers. What happens if you click "Open app"?

**Reply by alishahzad789** [reply]
code-server runs

**Reply by jsjoeio** [reply]
That's great! Does it give you a URL to access it?

**Reply by alishahzad789** [reply]
yes,it gives the url but i wanted to look in container

**Reply by jsjoeio** [reply]
Ahh! I just read docs. https://devcenter.heroku.com/articles/heroku-cli-commands#heroku-run

I'm not sure why this isn't working 🤔 Does Heroku have a community forum to ask in? Seems very odd
